anda/lib/qt5-qtdbustest/qt5-qtdbustest.spec

```
@@ -1,11 +1,14 @@
+%global forgeurl https://gitlab.com/ubports/development/core/libqtdbustest
+%global commit f204bb8d7e9a2ffcd159d2f51f47257a715bbd62
+%forgemeta
+
 Name:       qt5-qtdbustest
-Version:    0.2+bzr42
+Version:    0.3
 Release:    %autorelease
 Summary:    Library for testing DBus interactions using Qt5
 License:    LGPLv3
-URL:        https://launchpad.net/libqtdbustest
-Source0:    http://deb.debian.org/debian/pool/main/libq/libqtdbustest/libqtdbustest_0.2+bzr42+repack1.orig.tar.xz
-Source1:    https://salsa.debian.org/debian-ayatana-team/libqtdbustest/-/archive/master/libqtdbustest-master.tar.gz
+URL:        https://gitlab.com/ubports/development/core/libqtdbustest
+Source0:    %{url}/-/archive/%commit/qtdbustest-%commit.tar.gz
```

https://gitlab.com/ubports/development/core/libqtdbustest/-/archive/0.3.0/libqtdbustest-0.3.0.tar.gz
```suggestion
Source0:    %url/-/archive/%version/qtdbustest-%version.tar.gz
```